_maps/map_files/Deltastation/DeltaStation2.dmm

```diff
@@ -81858,66 +81854,53 @@
 	},
 /area/maintenance/port)
 "duw" = (
-/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden{
-	dir = 5
-	},
-/turf/open/floor/plasteel/vault{
-	dir = 5
-	},
+/turf/open/floor/engine,
 /area/science/explab)
 "dux" = (
-/obj/machinery/atmospherics/components/unary/vent_scrubber/on{
-	dir = 8
-	},
-/turf/open/floor/plasteel/vault{
-	dir = 5
-	},
+/obj/effect/turf_decal/bot_white,
+/obj/machinery/rnd/experimentor,
+/turf/open/floor/engine,
 /area/science/explab)
 "duy" = (
-/mob/living/simple_animal/pet/dog/pug{
-	name = "Swanson"
-	},
-/turf/open/floor/plasteel/vault{
-	dir = 5
-	},
-/area/science/explab)
-"duz" = (
 /obj/effect/turf_decal/bot_white,
-/turf/open/floor/plasteel/vault{
-	dir = 8
-	},
+/turf/open/floor/engine,
 /area/science/explab)
-"duA" = (
-/obj/machinery/rnd/experimentor,
-/obj/effect/decal/cleanable/dirt,
-/obj/effect/turf_decal/bot_white,
+"duz" = (
+/obj/effect/turf_decal/bot,
+/obj/machinery/airalarm/directional/west,
 /turf/open/floor/plasteel/vault{
 	dir = 8
 	},
-/area/science/explab)
+/area/science/nanite)
 "duC" = (
-/obj/machinery/atmospherics/pipe/simple/supply/hidden{
-	dir = 5
+/obj/effect/turf_decal/stripes/line{
+	dir = 8
 	},
 /turf/open/floor/plasteel/vault{
 	dir = 5
 	},
-/area/science/explab)
+/area/science/nanite)
 "duD" = (
-/obj/machinery/atmospherics/components/unary/vent_pump/on{
-	dir = 8
+/obj/structure/chair/office/light{
+	icon_state = "officechair_white";
```

Comment:
dv

if you wouldn't mind removing this from Delta, Box, Pubby, and Emergency_Cere, that'd be appreciated.

Reply:
O yeah, shouldn't take too long.